### PR TITLE
Patch: font style fixing and state refine for CANS-951 AssessmentDeleteModal

### PR DESCRIPTION
**Pull request by zliuwd**
## Description

- AssessmentDeleteModal related font size increased
- AssessmentDeleteModal state and cancel functionality refine
## Tests

- [ ] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by zliuwd** [comment]
retest this please

File: app/javascript/Application/components/Assessment/style.sass [hunk 1]
```diff
@@ -456,26 +457,26 @@ button.change-log-print-button
   width: 60%
 
 .delete-reason-textarea-empty
-  font-size: 1.2rem
+  font-size: 1.5rem
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/style.sass]: I am obligated to grumble about font-size changes like this, as they should ideally all be encapsulated in some shared Modal styles somewhere. For example, Modals in CANS should have the same styles as Modals in CALS or Intake (or even *other* Modals in CANS).

Anyways, I do not expect you to fix that. I am just continuing to raise awareness.

File: app/javascript/Application/components/Assessment/AssessmentHelper.js [hunk 2]
```diff
@@ -196,9 +196,9 @@ export const blankPlaceHolder = { value: '', label: '' }
 
 export const deleteWarningLabelStyle = {
   marginTop: '1.5rem',
-  marginBottom: '0.8rem',
+  marginBottom: '1rem',
```
Comment (nasar-cambria) [inline app/javascript/Application/components/Assessment/AssessmentHelper.js]: Is there a reason we are using a helper file for css and not importing it from style.sass ?

Comment (zliuwd) [reply app/javascript/Application/components/Assessment/AssessmentHelper.js]: Good catch + Good idea

File: app/javascript/Application/components/Assessment/style.sass (hunk 1 above)
Comment (zliuwd) [reply app/javascript/Application/components/Assessment/style.sass]: Great reminding! I created a global modal-styles.sass file to hold all of the style setting which related Modals. For current step I could make sure the style setting of original PageModal component and AssessmentDeleteModal are gathered into there, maybe other Modals still have some inline style, by our refactoring we will clean them up. Then every each work related Modals in the future we could set style for Modals more easily

Comment (zliuwd) [reply app/javascript/Application/components/Assessment/style.sass]: But for the "crossing project" shared Modal style, I think we still need some talking, maybe a story for creating something in design library.

**Pull request merged**